src/sentry/static/sentry/app/components/group/externalIssueForm.jsx

```diff
@@ -235,17 +241,17 @@ export class SentryAppExternalIssueForm extends React.Component {
       return '';
     }
 
+    this.model.setValue('action', this.props.action);
```

Comment:
What's the reason for changing out the `initialData`, it should accomplish the same thing

Reply:
`initialData` is only used once in the `constructor`, _not_ when a component is updated. The values can change on every re-render of this form component. The form wasn't picking up those changes because the constructor was never being called again.

Reply:
What's causing the component to update?

Reply:
It's rendered inside of a modal that has two tabs. When the User switches between the tabs, it changes the list of fields that get rendered, along with some of the data above like `action`, causing it update.

See: https://github.com/getsentry/sentry/pull/12558/files#diff-007d9ff646af0a6af90a199f2269623bR280